src/lib/MapCards.svelte

```diff
@@ -74,8 +80,8 @@
   {/if}
 
   {#if maps.length !== 0}
-    {#each maps as map}
-      <div class="card-wrapper">
+    {#each maps.slice(0, visibleCount) as map (map.id)}
+      <div class="card-wrapper" transition:slide={{ duration: 300 }}>
```

Comment:
Without it the cards just sort of pop-up out of nowhere and makes it feel more static. I figured this would be a good way to decrease the static appearance.

Reply:
Could be fine then.